.github/workflows/release.yml

```diff
@@ -10,7 +10,11 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: olafurpg/setup-scala@v14
+      - uses: actions/setup-scala@v3
```

Comment:
There's a typo here that broke release. it should be actions/setup-java
https://github.com/sbt/sbt-ci-release/actions/runs/5145242446/jobs/9262621172